webapp/components_handler.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2018 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package webapp
+
+import (
+	"fmt"
+	"io/ioutil"
+	"mime"
+	"net/http"
+	"path/filepath"
+	"regexp"
+
+	"github.com/gorilla/mux"
+)
+
+var packageRegex = regexp.MustCompile(`(import|from) (['"])(@[^/]*/)`)
```

Comment:
Since this is a dumb string replacer (instead of an AST walker), I'd rather having a stricter regex, perhaps only matching `import ... from @...` isnstead of accepting either `import` or `from` to minimize the risk of overwriting something else.